.github/scripts/lint-pr-title.cjs

```diff
@@ -0,0 +1,125 @@
+/**
+ * This Script can be tested locally with
+ *
+ * export GH_TOKEN=**** GITHUB_REPOSITORY=quisquous/cactbot PR_NUMBER=$NUM
+ * node ./.github/scripts/lint-pr-title.cjs
+ */
+'use strict';
+
+const github = require('@actions/github');
+
+const botName = 'cactbotbot';
+
+const validScope = [
+  'resources',
+  'ui',
+  'pullcounter',
+  'raidboss',
+  'raidemulater',
+  'oopsy',
+  'radar',
+  'eureka',
+  'jobs',
+  'test',
+  'util',
+  'config',
+  'lint',
+  'build',
+  'ci',
+  'i18n',
```

Comment:
Sorry for one more suggestion, but I sorted these into two categories and then alphabetized (so it's easier to find them).  I also added plugin and docs.  I did a search through `git log --oneline` and I think I caught everything.

```suggestion
  // UI Module Scopes
  'config',
  'dps',
  'eureka',
  'fisher',
  'jobs',
  'oopsy',
  'pullcounter',
  'radar',
  'raidboss',
  'raidemulater',

  // Other Scopes
  'build', // changes to webpack or TypeScript/babel toolchain
  'ci', // changes in .github/
  'docs', // documentation anywhere
  'i18n', // translations anywhere
  'lint', // eslint changes
  'plugin', // code in plugin/
  'resources', // code in resources/
  'test', // code in test/ or ui/test/
  'util', // code in util/
```